Sources/Jobs/JobQueue.swift

```diff
@@ -48,9 +48,9 @@ public struct JobQueue<Queue: JobQueueDriver>: Service {
         options: JobOptions = .init()
     ) async throws -> Queue.JobID {
         let buffer = try self.queue.encode(id: id, parameters: parameters)
-        Meter(label: "swift.jobs.meter", dimensions: [("status", "queued")]).increment()
         let jobName = id.name
         let id = try await self.queue.push(buffer, options: options)
+        Meter(label: "swift.jobs.meter", dimensions: [("status", "queued"), ("name", jobName)]).increment()
```

Comment:
I assume this was causing issues when a push fails?

Reply:
The meter label is hard coded here, but not in JobQueueHandler

Reply:
I am moving the labels into an internal structure named `JobMetricsHelper` so they can be accessible in the swift jobs library.